docs/administration.md

```diff
@@ -0,0 +1,49 @@
+# Administration
+
+You can acces uMap administration page by navigating to  `https://your.server.org/admin`
+
+You will have to connect with the admin account create during installation. Default admin username is umap.
+
+## Pictograms
+
+Pictograms are icon that can be used in your map markers.
+
+Pictograms are not embedded in uMap sources, you will have to add them manually. So you can choose which pictograms you want to use.
+
+Example of pictograms libraries you may want to use:
+
+- [Maki Icons](https://labs.mapbox.com/maki-icons/) (icon set made for map designers)
+- [Osmic Icons](https://gitlab.com/gmgeo/osmic)
+- [SJJB Icons](http://www.sjjb.co.uk/mapicons/contactsheet)
+
+### Import pictograms
+
+To import pictograms on your uMap server, you will need to use command `umap import_pictograms`
```

Comment:
Note: they can also be added one by one from the admin, even if it's a longer process it can help